lib/proximity.js

```diff
@@ -163,7 +176,15 @@ var Controllers = {
         // If the ADC output reads 700 the range in centimeters is 700 centimeters.
         //
         // ADC -> cm
-        return raw;
+        //
+        // Datasheet available at https://www.maxbotix.com/documents/XL-MaxSonar-EZ_Datasheet.pdf
+        //
+        // From intro on page 1
+        // 'The MB1200 and MB1300 sensor series detects objects from 0-cm1
+        //  to 765-cm (25.1 feet) or 1068cm (35 feet) (select models) and
+        // provide sonar range information from 20-cm2
+        //  out to765-cm or 1068-cm (select models) with 1-cm resolution...'
+        return toFixed(raw, 0);
```

Comment:
@rwaldron  Here I forced the call to `toFixed` even if not needed. Maybe it is too much. I can revert it if you prefer.

Reply:
How about an optimization? Truncate the decimal digits by shifting the bits off: `raw >> 0`, this shifts right by `0` bits, but first it must convert the left hand operand to an int32—which means discarding the exponent bits :)

Reply:
I'll do it and re open this PR, thanks!